pyproject.toml

```diff
@@ -83,8 +65,40 @@ git-email = "52496925+botpub@users.noreply.github.com"
 changelog-file = "docs/changelog.rst"
 changelog-header = "###############"
 version-header = "="
-version-strings = ["setup.py"]
-build-system = "setuptools"
+version-strings = ["pyproject.toml"]
```

Comment:
This line is not necessary. The `version-strings` AutoPub directive is only for _additional_ version strings that should be incremented, aside from the canonical one in `pyproject.toml`.

Reply:
This is marked as `resolved`, but the spurious line seems to still be present.